duneana/LowEAna/fcl/LowEAna.fcl

```diff
@@ -0,0 +1,68 @@
+BEGIN_PROLOG
+
+legacy_lowe_ana_hd:
+{
+  module_type:		     "LowEAna"
+  #====================================================================================
+  
+  RawDigitLabel:       "daq"		      # string for the process that made the raw digits
+  HitLabel:		         "hitfd" 	      # string for the process that made the reco hits
+  GEANT4Label:         "largeant"     # The label for the process which ran GEANT4
+  OpFlashLabel:        "opflash"
+  OpHitLabel:          "ophit"
+
+  # Vector with generator labels for the particles included in the event. IMPORTANT: Keep MARLEY label in the first position!!!
+  ParticleLabelVector: ["marley","ar39Gen","ar42Gen","apaGen","cNeutronGen","cpaGen","kr85Gen","rn222Gen","po210Gen"]
+  InteractionLabelVector: ["primary","nCapture"]
+  
+  #====================================================================================
+  # General geometry parameters
+  Geometry:            "HD"
+  DetectorSizeX:       365          
+  DetectorSizeY:       610           # Dont't change this value for HD unless you know what you are doing!!!
+
+  # Parameters for clustering algorithm
+  ClusterAlgoTime:       10          # Time window to look for hits in [tick] units.
+  ClusterAlgoAdjChannel: 2           # Number of adjacent channels to look for hits.
+  
+  # Parameters for the col vs ind plane matching
+  ClusterMatchNHit:     0.5          # DAQ Clusters min. hit requirement.
+  ClusterMatchCharge:   0.5          # Charge fraction to match clusters.
+  ClusterMatchTime:     20           # Time window to look for ind. plane clusters in [tick] units.
+  ClusterInd0MatchTime: 0            # Goal Time to match clusters in ind. plane 0 in [tick] units. Needs to be calibrated for each geometry!
+  ClusterInd1MatchTime: 0            # Goal Time to match clusters in ind. plane 1 in [tick] units. Needs to be calibrated for each geometry!
+  
+  ClusterPreselectionNHit: 2         # Number of hits to match clusters.
+  
+  # Parameters for the adjacent cluster matching
+  AdjClusterTime:      1200          # Time window to search for adj. clusters in [tick] units.
+  AdjClusterRad:       100           # Radius to search for adj. clusters in [cm] units.
+  
+  # Parameters for the adjacent OpFlash matching
+  AdjOpFlashTime:      4500          # Time window to look for adjacent flashes in [tick] units should be one full drift length.
+  AdjOpFlashRad:       200           # Radius to search for adj. OpFlashes reconstructed in [cm] units.
+  AdjOpFlashMaxPECut:  1.01          # Cut on the maximum PE OpHit contribution to the total OpFlash PE (lower limit).
```

Comment:
This looks strange to me. The maximum PE contribution is lower than the minimum PE cut? And why is there a maximum cut?

Reply:
Yes, you are right, the naming might not be super clear. There is a MinPE cut, and that one works as expected. The MaxPE cut is a cut on the ratio between the maximum number of PEs from an OpHit inside an OpFlash vs the total number of PEs of the flash. It's a way to cut on how "point-like" the flash is. But we should find a better name for it.